### PR TITLE
Issues 1 and 2

### PR DESCRIPTION
**Pull request by gdharra2**
vrticle -
Issue 1: Backend functionality implemented using command line inputs
Issue 2: Accept any folder structure
Test scripts implemented in test_functionality

### CODE REVIEW

File: utility.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+import zipfile
+import tempfile, shutil
+
+
+def get_input():
+    """
+
+    :return:
+    """
+    file_path = input('Enter the zip file path')
```
Comment (wallacerob) [inline utility.py]: format input with ": " or similar

File: test_functionality.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+from utility import validate_input
+
+
+def test_valid_input():
+    assert validate_input('tests/images_with_directory.zip',3,2)
+
+
+def test_invalid_input():
+    assert not validate_input('tests/images_without_directory.zip',4,2)
```
Comment (wallacerob) [inline test_functionality.py]: cleanup tmp dir after running tests

File: utility.py [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+        print('list of dir', zip_ref.namelist())
+
+        for member in zip_ref.namelist():
+            filename = os.path.basename(member)
+            # skip directories
+            if not filename:
+                continue
+
+            # copy file (taken from zipfile's extract)
+            print('member=',member)
```
Comment (wallacerob) [inline utility.py]: remove unnecessary print statements

File: utility.py (hunk 1 above)
Comment (gdharra2) [reply utility.py]: @wallacerob I have incorporated the changes. Please validated

File: utility.py (hunk 3 above)
Comment (gdharra2) [reply utility.py]: @wallacerob I have incorporated the changes. Please validated

**Pull request merged**